components/hubspot/sources/common.js

```diff
@@ -26,10 +26,15 @@ module.exports = {
     _setAfter(after) {
       this.db.set("after", after);
     },
+    _getProperties() {
+      return this.db.get("properties");
+    },
+    _setProperties(properties) {
+      this.db.set("properties", properties);
+    },
     async paginate(params, resourceFn, resultType = null, after = null) {
       let results = null;
-      let done = false;
-      while ((!results || params.after) && !done) {
+      while (!results || params.after) {
         results = await resourceFn(params);
         if (results.paging) params.after = results.paging.next.after;
```

Comment:
For better readability, I'd like to suggest you add the brackets in the statements
```suggestion
        if (results.paging) {
          params.after = results.paging.next.after;
        } else {
          delete params.after;
        }
        if (resultType) {
          results = results[resultType];
        }
```

Reply:
It would be great if you could do the same in `paginateUsingHasMore`, please